src/ui/printer.rs

```diff
@@ -62,7 +61,11 @@ impl<W: Write> Printer<W> {
                     );
                 } else {
                     let mut logo_lines = if let Some(custom_ascii) = &self.info.config.ascii_input {
-                        AsciiArt::new(custom_ascii, &colors, !self.info.config.no_bold)
+                        AsciiArt::new(
```

Comment:
This may be a little harder than that.
Indeed, `self.info.ascii_colors`'s capacity is based on the dominant language definition (look [here](https://github.com/o2sh/onefetch/blob/86f730791348357e9306d8b028a64341f1ed618e/src/ui/mod.rs#L9)). As a result, depending on the dominant language, output may differ:

Example 1 (shell has only 1 color):

<img width="719" alt="Screenshot 2022-06-12 at 01 23 25" src="https://user-images.githubusercontent.com/13710835/173208033-e5ce7c4a-ac8c-4d61-bb2e-8dad5302c2af.png">

Example 2 (TypeScript has 2 colors):

<img width="752" alt="Screenshot 2022-06-12 at 01 24 55" src="https://user-images.githubusercontent.com/13710835/173208076-cc2d9771-d7c5-4ea4-a3fa-35bed040052b.png">

Example 3 (c# has more than 3 colors):

<img width="703" alt="Screenshot 2022-06-12 at 01 25 52" src="https://user-images.githubusercontent.com/13710835/173208098-273cdb00-7c0a-4c77-9028-7a83224eb42f.png">

Reply:
Thank you for your guidance. I'll put a WIP prefix until I've made the requested changes.

Reply:
![image](https://user-images.githubusercontent.com/2551568/173230188-6a1713c0-bbe6-4b45-9700-e196b0f90419.png)

This handles your example cases.
Still rather new to rust so any criticism on the approach taken would be much appreciated.